general-superstaq/general_superstaq/check/all_.py

```diff
@@ -50,6 +38,7 @@ def run(*args: str, sphinx_paths: Optional[List[str]] = None) -> int:
 
     parser.add_argument(
         "-f",
+        "--soft-force",
         action="store_true",
         dest="force_formats",
```

Comment:
nit: for consistency can we get rid of this line, and either use `"--force-formats"` above or `parsed_args.soft_force` below?
```suggestion
```

Reply:
I made it `--force-formats` and `--force-all`.